sources/deny.toml

```diff
@@ -55,6 +55,8 @@ deny = [{ name = "structopt" }, { name = "clap", wrappers = ["cargo-readme"] }]
 skip = [
     # this can be removed once settings sdk is updated to base64 0.22.1
     { name = "base64", version = "=0.21.7" },
+    # this can be removed once settings sdk is updated to env_logger 0.11
```

Comment:
Can we have a PR with the update in the Bottlerocket Settings SDK? :D

Reply:
Already done! https://github.com/bottlerocket-os/bottlerocket-settings-sdk/pull/50